### PR TITLE
Henrynunez112/footer links

### LINKED ISSUE #92: Change Join to Careers in Footer
Make it simple for people looking for careers to find them. 

change footer link text from "join" to "Careers"

### PR DESCRIPTION
**Pull request by Henrynunez112**
**Description**
Changed Join to Careers on footer of celo.org

Screen Shot 2021-02-25 at 4 53 03 PM

**Tested**
Screen Shot 2021-02-25 at 5 08 46 PM

Fixes #92

### CODE REVIEW

**Review by aaronmgdr — APPROVED** [review]
nailed it 💯  🔨

File: src/shared/Footer.tsx [hunk 1]
```diff
@@ -118,6 +118,12 @@ export default function Footer({ hideForm }: Props) {
   const { t } = useTranslation(NameSpaces.common)
   const { isMobile, isTablet } = useScreenSize()
   const year = new Date().getFullYear()
+  const footerMenu = MENU.map((item) => {
+    return {
+      name: t(`footer.${item.name}`),
+      link: item.link 
+    }
+  })
```
Comment (aaronmgdr) [inline src/shared/Footer.tsx:126]:
```suggestion
const footerMenu = React.useMemo(() => MENU.map((item) => {
    return {
      name: t(`footer.${item.name}`),
      link: item.link 
    }
  }),[t] )
  ```

Comment (aaronmgdr) [reply src/shared/Footer.tsx:126]: avoid re running this map every render ;)

Comment (Henrynunez112) [reply src/shared/Footer.tsx]: 🔥🔥🔥🔥

**Pull request merged**